federation/pkg/kubefed/init/init.go

```diff
@@ -662,6 +669,7 @@ func createPVC(clientset client.Interface, namespace, svcName, federationName, e
 					api.ResourceStorage: capacity,
 				},
 			},
+			StorageClassName: storageClassName,
```

Comment:
Better to not set this field, if none is specified by user, so that `DefaultStorageClass` is used.
Here is note from the documentation in https://kubernetes.io/docs/concepts/storage/persistent-volumes/#storageclasses
- Do not use `volume.beta.kubernetes.io/storage-class`: with any value including the empty string since it will prevent DefaultStorageClass admission controller from running if enabled.

- In the future, we expect most clusters to have DefaultStorageClass enabled, and to have some form of storage available. However, there may not be any storage class names which work on all clusters, so continue to not set one by default. At some point, the alpha annotation will cease to have meaning, but the unset storageClass field on the PVC will have the desired effect.